kratos/includes/kratos_application.h

```diff
@@ -300,24 +300,24 @@ class KRATOS_API(KRATOS_CORE) KratosApplication {
     ///@{
     std::string mApplicationName;
 
-    //general conditions must be defined
+    // General conditions must be defined
```

Comment:
this file should not be modified in this PR

Reply:
OK